src/main/java/com/yscope/logging/log4j1/EnhancedAppenderSkeleton.java

```diff
@@ -1,34 +1,43 @@
 package com.yscope.logging.log4j1;
 
 import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.helpers.ISO8601DateFormat;
 import org.apache.log4j.helpers.LogLog;
 
+import java.util.Date;
+
 /**
  * {@code org.apache.log4j.AppenderSkeleton} enhanced with logging methods that
  * prepend the name of the logger.
  */
 public abstract class EnhancedAppenderSkeleton extends AppenderSkeleton {
   protected void logDebug (String msg) {
-    LogLog.debug("[" + name + "] " + msg);
+    LogLog.debug(getCurrentTimestampString() + " [" + name + "] " + msg);
   }
 
   protected void logDebug (String msg, Throwable t) {
-    LogLog.debug("[" + name + "] " + msg, t);
+    LogLog.debug(getCurrentTimestampString() + " [" + name + "] " + msg, t);
   }
 
   protected void logWarn (String msg) {
-    LogLog.warn("[" + name + "] " + msg);
+    LogLog.warn(getCurrentTimestampString() + " [" + name + "] " + msg);
   }
 
   protected void logWarn (String msg, Throwable t) {
-    LogLog.warn("[" + name + "] " + msg, t);
+    LogLog.warn(getCurrentTimestampString() + " [" + name + "] " + msg, t);
   }
 
   protected void logError (String msg) {
-    LogLog.error("[" + name + "] " + msg);
+    LogLog.error(getCurrentTimestampString() + " [" + name + "] " + msg);
   }
 
   protected void logError (String msg, Throwable t) {
-    LogLog.error("[" + name + "] " + msg, t);
+    LogLog.error(getCurrentTimestampString() + " [" + name + "] " + msg, t);
+  }
+
+  private static String getCurrentTimestampString() {
+    return timestampFormatter.format(new Date());
   }
+
+  private static final ISO8601DateFormat timestampFormatter = new ISO8601DateFormat();
```

Comment:
Variables should be at the top of the class (I know this disagrees with our C++ style guide, but other Java projects seem to follow this style).

Reply:
done